packages/serve/startDevServer.ts

```diff
@@ -0,0 +1,40 @@
+import * as Server from "webpack-dev-server/lib/Server";
+
+/**
+ *
+ * Starts the devServer
+ *
+ * @param {Object} compiler - a webpack compiler
+ * @param {Object} options - devServer options
+ * @param {Function} onListening - optional callback for when the server starts listening
+ * 
+ * @returns {Void} 
+ */
+export default function startDevServer(compiler, options, onListening): void {
+    const firstWpOpt = compiler.compilers
+        ? compiler.compilers[0].options
```

Comment:
Why only `0`? What about multi-mode compilation

Reply:
@evilebottnawi In the past on `webpack-dev-server` we only looked at the first compiler in a multi-compiler array to see if it has the `devServer` object. Do you want to change this behavior?